firmware/ec/test/fake/fake_SX1509_register.c

```diff
@@ -0,0 +1,123 @@
+    [0x33] = 0x00, /* ON intensity register I/O[3] */
+    [0x34] = 0x00, /* OFF time/intensity register I/O[3] */
+    [0x35] = 0x00, /* ON time register I/O[4] */
+    [0x36] = 0x00, /* ON intensity register I/O[4] */
+    [0x37] = 0x00, /* OFF time/intensity register I/O[4] */
+    [0x38] = 0x00, /* Fade in register I/O[4] */
+    [0x39] = 0x00, /* Fade out register I/O[4] */
+    [0x3A] = 0x00, /* ON time register I/O[5] */
+    [0x3B] = 0x00, /* ON intensity register I/O[5] */
+    [0x3C] = 0x00, /* OFF time/intensity register I/O[5] */
+    [0x3D] = 0x00, /* Fade in register I/O[5] */
+    [0x3E] = 0x00, /* Fade out register I/O[5] */
+    [0x3F] = 0x00, /* ON time register I/O[6] */
+    [0x40] = 0x00, /* ON intensity register I/O[6] */
+    [0x41] = 0x00, /* OFF time/intensity register I/O[6] */
+    [0x42] = 0x00, /* Fade in register I/O[6] */
+    [0x43] = 0x00, /* Fade out register I/O[6] */
+    [0x44] = 0x00, /* ON time register I/O[6] */
+    [0x45] = 0x00, /* ON intensity register I/O[7] */
+    [0x46] = 0x00, /* OFF time/intensity register I/O[7] */
+    [0x47] = 0x00, /* Fade in register I/O[7] */
+    [0x48] = 0x00, /* Fade out register I/O[7] */
+    [0x49] = 0x00, /* ON time register I/O[8] */
+    [0x4A] = 0x00, /* ON intensity register I/O[8] */
+    [0x4B] = 0x00, /* OFF time/intensity register I/O[8]  */
+    [0x4C] = 0x00, /* ON time register I/O[9] */
+    [0x4D] = 0x00, /* ON intensity register I/O[9] */
+    [0x4E] = 0x00, /* OFF time/intensity register I/O[9] */
+    [0x4F] = 0x00, /* ON time register I/O[10] */
+    [0x50] = 0x00, /* ON intensity register I/O[10] */
+    [0x51] = 0x00, /* OFF time/intensity register I/O[10] */
+    [0x52] = 0x00, /* ON time register I/O[11] */
+    [0x53] = 0x00, /* ON intensity register I/O[11] */
+    [0x54] = 0x00, /* OFF time/intensity register I/O[11] */
+    [0x55] = 0x00, /* ON time register I/O[12] */
+    [0x56] = 0x00, /* ON intensity register I/O[12] */
+    [0x57] = 0x00, /* OFF time/intensity register I/O[12] */
+    [0x58] = 0x00, /* Fade in register I/O[12] */
+    [0x59] = 0x00, /* Fade out register I/O[12] */
+    [0x5A] = 0x00, /* ON time register I/O[13] */
+    [0x5B] = 0x00, /* ON intensity register I/O[13] */
+    [0x5C] = 0x00, /* OFF time/intensity register I/O[13] */
+    [0x5D] = 0x00, /* Fade in register I/O[13] */
+    [0x5E] = 0x00, /* Fade out register I/O[13] */
+    [0x5F] = 0x00, /* ON time register I/O[14] */
+    [0x60] = 0x00, /* ON intensity register I/O[14] */
+    [0x61] = 0x00, /* OFF time/intensity register I/O[14] */
+    [0x62] = 0x00, /* Fade in register I/O[14] */
+    [0x63] = 0x00, /* Fade out register I/O[14] */
+    [0x64] = 0x00, /* ON time register I/O[15] */
+    [0x65] = 0x00, /* ON intensity register I/O[15] */
+    [0x66] = 0x00, /* OFF time/intensity register I/O[15] */
+    [0x67] = 0x00, /* Fade in register I/O[115] */
+    [0x68] = 0x00, /* Fade out register I/O[15] */
+    [0x69] = 0x00, /*  */
+    [0x6A] = 0x00, /*  */
```

Comment:
This looks to be a direct copy of the register map addresses from the SX1509 datasheet, so might be good to just add a comment specifying that to make things clear? I don't think enum would serve the purpose here, since this acts as the device memory for unit tests.

Reply:
Agree, This is taken from SX1509 datasheet. Will add comment to specify that no data is available in the datasheet for registers [0x6B] thru [0x7C].